core/base_service.go

```diff
@@ -289,6 +291,7 @@ func (service *BaseService) DisableSSLVerification() {
 		// Disable server ssl cert & hostname verification.
 		tr.TLSClientConfig.InsecureSkipVerify = true // #nosec G402
 	}
+	GetLogger().Debug("Disabled SSL verification in HTTP client")
```

Comment:
This message seems helpful but I don't think I remember seeing this in the other cores (or at least, not in Node). Perhaps we should add it there as well? (Maybe I missed it)

Reply:
yeah, it turns out that we have no `disableSslVerification()` in the node core's BaseService class.

Reply:
Oh, interesting. I think there is in the authenticators in the Node core. Odd that we aren't consistent there. Perhaps we should be, but that can be a follow-on effort.